apps/ios/Shared/Views/Chat/ChatView.swift

```diff
@@ -462,7 +462,7 @@ struct ChatView: View {
         private func menu() -> [UIAction] {
             var menu: [UIAction] = []
             if let mc = ci.content.msgContent, !ci.meta.itemDeleted || revealed {
-                if !ci.meta.itemDeleted {
+                if !ci.meta.itemDeleted && ci.id != ChatItem.TEMP_LIVE_CHAT_ITEM_ID {
```

Comment:
using special ID for an item to indicate that it is special is a hack, a better approach is to have a flag that is not included in JSON.